snapstudysensei/windows_list/__init__.py

```diff
@@ -0,0 +1,6 @@
+import sys
+
+if sys.platform.startswith("win"):
+    from .windows_list_win32 import WindowsList
+elif sys.platform.startswith("linux"):
```

Comment:
Can you make that a simple `else`? BSD & friends might be happy with the X11 support